ghost/core/core/server/data/schema/schema.js

```diff
@@ -413,7 +413,8 @@ module.exports = {
         reason: {type: 'string', maxlength: 50, nullable: true},
         feature_image: {type: 'string', maxlength: 2000, nullable: true},
         feature_image_alt: {type: 'string', maxlength: 191, nullable: true, validations: {isLength: {max: 125}}},
-        feature_image_caption: {type: 'text', maxlength: 65535, nullable: true}
+        feature_image_caption: {type: 'text', maxlength: 65535, nullable: true},
+        custom_excerpt: {type: 'string', maxlength: 2000, nullable: true, validations: {isLength: {max: 300}}}
```

Comment:
Any reason to not just use `string` as the type? (`posts.custom_excerpt` uses `string`)

Reply:
Not too sure I'm following this. `custom_excerpt` has `type: 'string'` here too - it's copied directly from the posts table in the schema.

Reply:
Ah my bad, i commented on the wrong line - It was [here](https://github.com/TryGhost/Ghost/pull/20323/files#diff-07ed1b3460b36fa1a09f53c5b03155fab9408cc106dc1f1cf72ba3bae965f6a7R4)

Reply:
Ohh, good catch, thanks!